src/test/java/racingcar/step3/CarTest.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.step3;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CarTest {
+
+    private Race race;
+
+    @DisplayName("지정된 수의 Car 객체를 cars 목록을 초기화")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 4, 7, 16, 20})
+    public void move(int input) {
+        race = new Race(input);
+        List<Car> cars = race.getCars();
+        assertEquals(input, cars.size(), "입력된 차량 갯수와 생성된 차량 걋수가 일치하지 않습니다.");
```

Comment:
1, 2단계에서는 assertj의 `assertThat()`을 사용했는데 여기서는 jupiter를 사용했네요
jupiter 보다는 assertj를 사용하는 것을 권장드립니다.
assertj의 assertThat은 체이닝 메소드를 제공하고 있어 메소드명을 외우지 않아도되고 가독성도 좋습니다.
```suggestion
        assertThat(cars).hasSize(expected);
```